lib/facter/resolvers/memory_resolver.rb

```diff
@@ -48,7 +48,12 @@ def kilobytes_to_bytes(quantity)
           def reclaimable_memory(output)
             buffers = kilobytes_to_bytes(output.match(/Buffers:\s+(\d+)\s/)[1])
             cached = kilobytes_to_bytes(output.match(/Cached:\s+(\d+)\s/)[1])
-            s_reclaimable = kilobytes_to_bytes(output.match(/SReclaimable:\s+(\d+)\s/)[1])
+            s_reclaimable = output.match(/SReclaimable:\s+(\d+)\s/)
```

Comment:
I think that we could use something like this: s_reclaimable = output.match(/SReclaimable:\s+(\d+)\s/).fetch(1) { 0 }

Reply:
will not solve the problem : `undefined` method 'fetch' for nil:NilClass`.
Also if we do : `&.fetch(1) { 0 }` the result will be nil

Reply:
yes, you are right